package.json

```diff
@@ -11,8 +11,8 @@
     "gendoc": "jsdoc -r lib -R README.md -P package.json -d .jsdoc",
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "eslint -c .eslintrc --max-warnings 0 src/**/*.ts",
-    "prepublishOnly": "npm run build",
-    "build": "tsc --project tsconfig.json"
+    "build": "tsc --project tsconfig.json",
+    "prepare": "npm run build"
```

Comment:
This has also changed. This means the package should build on local install (only) automatically, rather than just publish.